tpot/base.py

```diff
@@ -286,7 +286,7 @@ def fit(self, features, classes):
         features = features.astype(np.float64)
 
         # Set the seed for the GP run
-        if self.random_state:
+        if self.random_state is not None:
```

Comment:
I went to look up what the difference was. According to this thread hey compile to the same bytecode: http://stackoverflow.com/questions/2710940/python-if-x-is-not-none-or-if-not-x-is-none